src/dataSources/cloudFirestore/session.js

```diff
@@ -25,6 +26,34 @@ const session = dbInstance => {
     return results;
   }
 
+  async function findAllApprovedByEventIdAtDate(eventId, atDate, daysAfter) {
+    dlog(
+      'findAllApprovedByEventIdAtDate(eventId, atDate, daysAfter)',
+      eventId,
+      atDate,
+      daysAfter,
+    );
+    let query = sessionsCollections
+      .where('eventId', '==', eventId)
+      .where('status', 'in', approvedSessionStatuses);
+
+    if (atDate) {
+      const fromdate = new Date(atDate);
+      query = query.where('startTime', '>=', atDate);
+
+      if (daysAfter) {
+        // 24 * 60 * 60 * 1000 === 86400000
+        const todate = new Date(fromdate.getTime() + daysAfter * 86400000);
```

Comment:
Couldn't you just do something like this https://day.js.org/docs/en/manipulate/add just take the time given, add the days and then call getTime?

Reply:
I was using native over pulling in another package. I guess we have Notion on the project already and I can use that if it makes it more readable overall. @csell5